source/containers.tex

```diff
@@ -2428,6 +2459,16 @@
   \end{itemize}
   where \tcode{r1} and \tcode{r2} are keys of elements in \tcode{a_tran},
 \item
+\tcode{kx} is a value such that
+  \begin{itemize}
+  \item \tcode{eq(r1, kx) == eq(kx, r1)},
+  \item \tcode{hf(r1) == hf(kx)} if \tcode{eq(r1, kx)} is \tcode{true},
+  \item \tcode{(eq(r1, kx) \&\& eq(r1, r2)) == eq(r2, kx)}, and
```

Comment:
Is this condition right? Can't `eq(r2, kx)` hold, but `eq(r1, r2)` and `eq(r1, rx)` could be false?